operate/client/src/modules/stores/processInstanceListeners.test.ts

```diff
@@ -80,8 +80,6 @@ describe('stores/processInstancesListeners', () => {
   });
 
   it('should fetch next listeners', async () => {
-    mockFetchProcessInstanceListeners().withSuccess(mockListenerInstances);
-
```

Comment:
This is unrelated to this issue, I just noticed we already do the same thing on the `beforeEach` hook for this test suite, so this is unnecessary. Same situation for lines 147-148.